Sdk.zig

```diff
@@ -32,14 +32,11 @@ prepare_sources: *PrepareStubSourceStep,
 
 /// Creates a instance of the Sdk and initializes internal steps.
 /// Initialize once, use everywhere (in your `build` function).
-pub fn init(b: *Builder) *Sdk {
+pub fn init(b: *Builder, config_path: []const u8) *Sdk {
```

Comment:
Can you make `config_path` optional, so people can just pass `null` for the old behaviour?